.editorconfig

```diff
@@ -1,30 +0,0 @@
-root = true
-
-[*]
-charset = utf-8
-end_of_line = lf
-insert_final_newline = true
-trim_trailing_whitespace = true
-indent_size = 4
```

Comment:
from: https://prettier.io/docs/options
> Setting indent_size or tab_width in an [.editorconfig file](https://editorconfig.org/) will configure Prettier’s tab width, unless overridden.

so we need to update the `.prettierrc`

Reply:
thanks!